src/main/kotlin/net/atos/zac/zaak/ZaakService.kt

```diff
@@ -23,70 +24,66 @@ import net.atos.client.zgw.zrc.model.Zaak
 import net.atos.client.zgw.ztc.ZtcClientService
 import net.atos.client.zgw.ztc.model.generated.OmschrijvingGeneriekEnum
 import net.atos.client.zgw.ztc.model.generated.RolType
+import net.atos.zac.app.klant.model.klant.IdentificatieType
 import net.atos.zac.event.EventingService
 import net.atos.zac.identity.model.Group
 import net.atos.zac.identity.model.User
 import net.atos.zac.websocket.event.ScreenEventType
 import nl.lifely.zac.util.AllOpen
+import java.util.EnumSet
+import java.util.Locale
 import java.util.UUID
 import java.util.logging.Logger
 
 private val LOG = Logger.getLogger(ZaakService::class.java.name)
 
 @AllOpen
+@Suppress("TooManyFunctions")
 class ZaakService @Inject constructor(
     private val zrcClientService: ZrcClientService,
     private val ztcClientService: ZtcClientService,
     private var eventingService: EventingService,
 ) {
-    fun addBetrokkenNatuurlijkPersoon(
-        roltype: RolType,
-        bsn: String,
-        zaak: Zaak,
-        toelichting: String
-    ) {
-        zrcClientService.createRol(
-            RolNatuurlijkPersoon(
-                zaak.url,
-                roltype,
-                toelichting,
-                NatuurlijkPersoon(bsn)
-            ),
-            toelichting
-        )
+    companion object {
+        val ZAAK_BETROKKENEN_ENUMSET: EnumSet<OmschrijvingGeneriekEnum> =
```

Comment:
Better to pull this up into it's own class or file?

Reply:
Yeah true. Was in doubt where to move this. It kind of belongs to the client packages but also not really.. It's also very weird that currently also the KlantRestService uses this. That's because the KlantRestService exposes some very weird endpoints which really have nothing at all to do with 'klanten' (but that's for another day..).